tests/topotests/ldp-oc-topo1/test_ldp_oc_topo1.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+
+#
+# test_ldp_oc_topo1.py
+# Part of NetDEF Topology Tests
+#
+# Copyright (c) 2020 by
+# Network Device Education Foundation, Inc. ("NetDEF")
```

Comment:
who wrote this again?

Reply:
Fixed copyright comment to indicate: Volta Newtorks.